linux-tools/test_time/test_time.py

```diff
@@ -0,0 +1,196 @@
+#!/bin/python
+
+from autotest.client import test, utils
+from distutils.spawn import find_executable
+from autotest.client.shared import error
+
+
+class test_time(test.test):
+    """
+    Autotest module for time utility.
+    time - time is a simple command,it gives resource usage.
+    The time utility runs the specified program/command with the given arguments.When program/command finishes,
```

Comment:
s/arguments./arguments. /
It's better to limit the line length to <80, for all of the file